src/sentry/api/event_search.py

```diff
@@ -47,14 +47,16 @@
     r"""
 search = spaces term*
 
-term = (boolean_operator / paren_group / filter / free_text) spaces
+term = (boolean_operator / paren_group / filter / free_email_text / free_text) spaces
 
 boolean_operator = or_operator / and_operator
 
 paren_group = open_paren spaces term+ closed_paren
 
+free_email_text = ~r"[a-zA-Z0-9*_.+-]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+"
```

Comment:
Wonder if it would be worth extracting the regex to a constant like you had before so we could have unit tests on it -- the tests were helpful to see what we expected to match/not match. Thoughts on this @JoshFerge?

Reply:
(I think this can be done in a follow-up if we decide to pursue)

Reply:
makes sense to me.